index.html

```diff
@@ -1,177 +1,241 @@
-  <!DOCTYPE html>
+<!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css"><link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
     <title>My Portfolio</title>
-</head>
+  </head>
 
-<body>
+  <body>
     <header id="page-header">
-        <a href="" id="my-logo" class="blue-text">My logo</a>
-        <nav>
-            <img src="icons/Hamburger-menu.png" alt="">
-        </nav>
+      <a href="" id="my-logo" class="blue-text">My logo</a>
+      <nav>
+        <img src="icons/Hamburger-menu.png" id="hamburger-menu" />
+        <ul id="header-ul">
+          <li class="header-li">
+            <a href="header-link">Portfolio</a>
+          </li>
+          <li class="header-li">
+            <a href="header-link">About</a>
+          </li>
+          <li class="header-li">
+            <a href="header-link">Contact</a>
+          </li>
+        </ul>
```

Comment:
- [ ] Your navbar is good and very similar to the Figma template. But kindly reduce the font-size of the items to match the Figma template and look more professional.
 - [ ] Also please use the proper id of the sections in the `href` attribute of the nav items to redirect the user to the correct part of the page. Currently, this is making the user leave the page and this is not a good UX.